examples/jsonplaceholder.graphql

```diff
@@ -1,11 +1,17 @@
 schema
   @server(port: 8000, enableGraphiql: true, enableQueryValidation: false, hostname: "0.0.0.0")
-  @upstream(baseURL: "http://jsonplaceholder.typicode.com", enableHttpCache: true) {
+  @upstream(baseURL: "http://jsonplaceholder.typicode.com", enableHttpCache: true)
+  @js_plugin(src: "target/debug") {
```

Comment:
```suggestion
  @plugin(name: "js", src: "target/debug") {
```

We should have a generic directive to import a plugin. We should also add a name, which will help us a bit in tracing and logging later.

Is the path is relative to the `tc` command?

Reply:
yeah, that's still much to change in terms of usage. I'm definitely going to improve how the plugin is resolver, its setup and ofc put it in separate file for testing. The most of current configuration I was using for quick experimenting